### PR TITLE
feat: add VM.get/setLabels() methods

### LINKED ISSUE #116: setLabels method feature request
*From customer:*

We use cloud function to label all of the instances created in our projects with the user id of the person who created it. This way we can notify the user when their instance comes under governance. Basically we create a sink from the gce instance log and filter it to look at insert events. This gets written to a bucket (or a pub/sub topic) that triggers a cloud function to run.
 

Here is what our instance tagging cloud function looks like.

 

exports.processFile = function(event, callback) {

    // Requires

    const path = require('path');

    const os = require('os');

    const fs = require('fs');

    const readline = require('readline');

    var storage = require('@google-cloud/storage')();

    var google = require('googleapis');

    var compute = google.compute('beta');

 

That obviously changed since the compute is now v1. But if I try either I get the following error.

2018-05-14 08:18:08.522 EDTnetapp-hcl-func 101128355950845 TypeError: google.compute is not a function at exports.processFile (/user_code/index.js:19:27) at 

 

So I try this one that gives me almost everything I need.

const Compute = require('@google-cloud/compute');

const compute = new Compute();

 

Except there is no setLabels() API anywhere to be found in that Node.js package.

### PR DESCRIPTION
**Pull request by jpatokal**
feat: add VM.get/setLabels() methods

Implemented using VM.get/setTags() as a template.

Fixes #116

### CODE REVIEW

**Comment by stephenplusplus** [comment]
Thank you very much! The only thing we need to merge is a system test to confirm it works when run against the real backend. Would you be willing to add those to "system-test/compute.js"? You'd have to test against your own project, then you can run them with `npm run system-test` from the root directory. No worries if not, just let me know!

Thanks again for helping out 👍🏻

**Review by stephenplusplus — CHANGES_REQUESTED** [review]
Perfect! Just need system tests before we can merge.

**Comment by jpatokal** [comment]
System tests added.  Instead of adding a new describe block, I placed it in the existing vm block, so we don't need to spin up a new VM just for this.

Note that the setLabels method requires explicitly passing in the label fingerprint.  IMHO this is a little unintuitive, but it parallels the underlying API and the existing setTags method as well.

File: system-test/compute.js [hunk 1]
```diff
@@ -934,7 +934,7 @@ describe('Compute', () => {
     });
   });
 
-  describe('vms', () => {
+  describe.only('vms', () => {
```
Comment (bcoe) [inline system-test/compute.js:937]: we should make sure we remove this `.only`.

Comment (jpatokal) [reply system-test/compute.js]: Fixed.

**Review by stephenplusplus — APPROVED** [review]

**Comment by stephenplusplus** [comment]
Looks perfect, thank you!

**Pull request merged**